src/api.js

```diff
@@ -104,6 +104,19 @@ const endpoints = {
   },
   allVulnerabilitiesForRepo: (name) =>
     `/v2/_zot/ext/search?query={CVEListForImage(image: "${name}"){Tag Page {TotalCount ItemCount} CVEList {Id Title Description Severity Reference PackageList {Name InstalledVersion FixedVersion}}}}`,
+  cveDiffForImages: (minuend = {}, subtrahend = {}, { pageNumber = 1, pageSize = 3 }) => {
+    let imageInput = (img) => {
+      let digest = img.digest ? `, digest: ${img.digest}` : '';
+      let platform = img.platform ? `, platform: {os: ${img.platform.os}, arch: ${img.platform.arch}}` : '';
+      return `{repo: "${img.repo}", tag: "${img.tag}"${digest}${platform}}`;
+    };
+    let query = `/v2/_zot/ext/search?query={CVEDiffListForImages(minuend: ${imageInput(
```

Comment:
no need for the 'query' variable, we can just return the interpolated string directly